scripts/install.sh

```diff
@@ -182,6 +182,10 @@ function init() {
         mv -f /etc/kubernetes/pki/apiserver.key /tmp/
     fi
 
+    # ensure that /etc/kubernetes/audit.yaml exists
+    cp $kustomize_kubeadm_init/audit.yaml /etc/kubernetes/audit.yaml
+    mkdir -p /etc/kubernetes/auditlog
```

Comment:
the above two lines need to be swapped? also the destination dir path is `/etc/kubernetes/auditlog/audit.yaml`

Reply:
no, they're two different things - the mkdir is for where we're logging to, and the audit.yaml is the spec of what to log